server/importers/index.ts

```diff
@@ -1,21 +1,55 @@
-const fs = require('fs')
-const path = require('path')
-const rimraf = require('rimraf')
-const sql = require('mssql')
-
-const log = require('../logger.js')
-const GtfsImport = require('../db/gtfs-import.js')
-const CreateShapes = require('../db/create-shapes.js')
-const connection = require('../db/connection.js')
-const Storage = require('../db/storage.js')
-const KeyvalueDynamo = require('../db/keyvalue-dynamo.js')
+import { join } from 'path'
+import { VarChar } from 'mssql'
+import { red } from 'colors'
+import log from '../logger.js'
+import GtfsImport from '../db/gtfs-import.js'
+import connection from '../db/connection.js'
+import Storage from '../db/storage.js'
+import KeyvalueDynamo from '../db/keyvalue-dynamo.js'
+import config from '../config'
+
+import ATImporter from './regions/nz-akl'
+import ChchImporter from './regions/nz-chc'
+import OtagoImporter from './regions/nz-otg'
+import TCImporter from './regions/au-cbr'
+import TfNSWImporter from './regions/au-syd'
+import MetlinkImporter from './regions/nz-wlg'
+import PTVImporter from './regions/au-mel'
+import RATPImporter from './regions/fr-par'
+import SEQImporter from './regions/au-seq'
+import SBBCFFFFSImporter from './regions/ch-sfr'
+import BaseImporter from './BaseImporter.js'
+import MultiImporter from './MultiImporter.js'
+
+const regions = {
+  'nz-akl': ATImporter,
```

Comment:
as cool as the transit agencies are, it's a little confusing to use them. Maybe we should just use names like NzAklImporter or AuSydImporter as opposed to AT or TfNSW?